project/Doc.scala

```diff
@@ -120,7 +120,10 @@ object UnidocRoot extends AutoPlugin {
 
   override lazy val projectSettings =
     CliOptions.genjavadocEnabled.ifTrue(scalaJavaUnidocSettings).getOrElse(scalaUnidocSettings) ++
-    settings(Seq(), Seq(AkkaBuild.remoteTests, AkkaBuild.benchJmh, AkkaBuild.protobuf, AkkaBuild.akkaScalaNightly, AkkaBuild.docs))
+      // TODO These should be excluded, but I haven't figured out how to exclude them without having the project
+      //   reference available anymore (since the Project's are in build.sbt now)
+      // AkkaBuild.remoteTests, AkkaBuild.benchJmh, AkkaBuild.protobuf, AkkaBuild.akkaScalaNightly, AkkaBuild.docs))
+    settings(Seq(), Seq())
```

Comment:
Such references are a final leftover of the old build structure. This code was migrated to an AutoPlugin but not completely. To finish the migration you should define a setting key in this autoplugin:
```scala
object autoImport {
  val unidocRootIgnoreProjects = settingKey[Seq[Project]]("Projects to ignore when generating unidoc")
}
import autoImport._
```
Then use the value of this setting in the autoplugin. And lastly set the setting value in the root project in the build.sbt
```scala
lazy val root = Project(
     id = "akka",
     base = file("."),
     aggregate = aggregatedProjects)
  .settings(rootSettings: _*)
  .settings(unidocRootIgnoreProjects := Seq(remoteTests))
```